components/brave_news/browser/resources/OptIn.tsx

```diff
@@ -57,22 +58,26 @@ const Container = styled.div`
   }
 `
 
-const description = formatLocale('braveNewsIntroDescriptionTwo', {
-  $1: (content) => (
-    <SecureLink href='https://brave.com/privacy/browser/'>{content}</SecureLink>
-  )
-})
-
 export default function OptIn() {
   const { toggleBraveNewsOnNTP } = useBraveNews()
 
   return (
-    <Container data-theme='dark'>
+    <Container data-theme='dark' className={NEWS_FEED_CLASS}>
       <div className='graphic' />
       <h3>{getLocale('braveNewsIntroTitle')}</h3>
       <div>
         <p>{getLocale('braveNewsIntroDescription')}</p>
-        <p>{description}</p>
+        <p>
+          {
+            formatLocale('braveNewsIntroDescriptionTwo', {
```

Comment:
I think you should leave this outside the function so it only gets evaluated once (it never changes)

Reply:
I'm currently not loading the news strings into storybook, and at the module level this `formatLocale` call will immediately throw an error. If we need this at the module level, I can start adding the news strings to storybook.

Reply:
Hmm - maybe just add this one to the Storybook for now?

Reply:
done